src/main/java/seedu/sherpass/util/Reminder.java

```
@@ -41,44 +41,52 @@ public void showReminders() {
      * with reference to user local machine date.
      */
     public void showDailyTask() {
-        ArrayList<Task> filteredDailyTasks = (ArrayList<Task>) tasks.stream()
-                .filter((t) -> isEqualDate(t.getByDate(), currentDate))
-                .filter((t) -> !t.isDone())
-                .collect(toList());
+        try {
+            ArrayList<Task> filteredDailyTasks = (ArrayList<Task>) tasks.stream()
+                    .filter((t) -> isEqualDate(t.getByDate(), currentDate))
+                    .filter((t) -> !t.isDone())
+                    .collect(toList());
 
-        if (filteredDailyTasks.isEmpty()) {
-            ui.showToUser("Your schedule is empty today.");
-        } else {
-            System.out.println("Schedule for today:");
-            for (Task task : filteredDailyTasks) {
-                System.out.println(task.toString());
+            if (filteredDailyTasks.isEmpty()) {
+                ui.showToUser("Your schedule is empty today.");
+            } else {
+                System.out.println("Schedule for today:");
+                for (Task task : filteredDailyTasks) {
+                    System.out.println(task.toString());
+                }
             }
+            ui.showLine();
+        } catch (Exception e) {
```

What is the exception being caught here?

---

It is a temporary fix to catch all exceptions to prevent the application from crashing. I know it is a bad practice. I will have a proper fix later.